ui-tests-protractor/e2e/integrations/twitter-to-salesforce-integration.feature

```diff
@@ -65,6 +68,15 @@ Feature: Create integration with datamapper step
     And scroll "top" "right"
     And click on the "Done" button
 
+    # add a filter step
+    When Camilla clicks on the "Add a Step" button
+    Then Camilla is presented with the "Add a step" link
+    And clicks on the random "Add a step" link
+    And she selects "Basic Filter" integration step
+    And she is presented with a "Basic Filter" step configure page
+    Then she fill configure page for "Basic Filter" step with "ANY of the following, path2, Contains, value2" parameter
```

Comment:
Let's be more realistic and use something like `text` contains `#salute` or `#syndesis4ever`.

Reply:
Hashtag part is especially important to mimic Twitter-like behaviour.

Reply:
OK